src/main/kotlin/domain/CoordinateRandomGenerator.kt

```diff
@@ -0,0 +1,12 @@
+package domain
+
+import dto.CoordinateGeneratorRequest
+
+class CoordinateRandomGenerator(coordinateGeneratorRequest: CoordinateGeneratorRequest) : CoordinateGenerator {
+    private val coordinates = coordinateGeneratorRequest.coordinates
+    private val landMine = coordinateGeneratorRequest.landMine
+
+    override fun coordinates(): List<Coordinate> {
+        return coordinates.shuffled().subList(0, landMine)
```

Comment:
`subList` 를 사용하면서 `0`이라는 불필요한 값을 사용해야하는것 같아 보입니다. `take`를 활용해보시는건 어떨까요?
```suggestion
        return coordinates.shuffled().take(landMine)
```